packages/api-reference/src/components/Content/Introduction/Introduction.vue

```diff
@@ -23,33 +23,48 @@ const props = defineProps<{
   parsedSpec: Spec
 }>()
 
-const slugger = new GithubSlugger()
-
-const specVersion = computed(() => {
-  return props.parsedSpec.openapi ?? props.parsedSpec.swagger ?? ''
+/**
+ * Get the OpenAPI/Swagger specification version from the API definition.
+ */
+const oasVersion = computed(() => {
+  return props.parsedSpec?.openapi ?? props.parsedSpec?.swagger ?? ''
 })
 
-const formattedSpecTitle = computed(() => {
-  return slugger.slug(props.info.title ?? '')
+/**
+ * Format the title to be displayed in the badge.
+ *
+ * TODO: We should move this logic to the DownloadLink component
+ */
+const slugger = new GitHubSlugger()
+const filenameFromTitle = computed(() => slugger.slug(props.info?.title ?? ''))
+
+/** Format the version number to be displayed in the badge */
+const version = computed(() => {
+  // Prefix the version with “v” if the first character is a number, don’t prefix if it’s not.
+  // Don’t output anything when version is not a string.
+  return typeof props.info?.version === 'string'
+    ? props.info.version.toString().match(/^\d/)
+      ? `v${props.info.version}`
+      : props.info.version
+    : undefined
```

Comment:
```suggestion
props.info?.version?.toString().startsWith('0-9') ? `v${props.info.version}` : props.info?.version
```
this version will handle numbers instead of returning undefined. completely up to you though!

Reply:
`startsWith(0-9)` checks for the literal `0-9`, not for numbers between 0 and 9 😬 

and without the string check it would output arrays, objects, booleans … (which no one should pass, but doesn’t hurt to add the check)